AsyncDisplayKit/ASTableView.mm

```diff
@@ -876,6 +876,33 @@ - (ASCellNode *)dataController:(ASDataController *)dataController nodeAtIndexPat
   return node;
 }
 
+- (ASDataControllerCellNodeBlock)dataController:(ASDataController *)dataController nodeBlockAtIndexPath:(NSIndexPath *)indexPath {
+  if (![_asyncDataSource respondsToSelector:@selector(tableView:nodeBlockForRowAtIndexPath:)]) {
+    ASCellNode *node = [_asyncDataSource tableView:self nodeForRowAtIndexPath:indexPath];
+    return ^{
+      [node enterHierarchyState:ASHierarchyStateRangeManaged];
+      ASDisplayNodeAssert([node isKindOfClass:ASCellNode.class], @"invalid node class, expected ASCellNode");
+      if (node.layoutDelegate == nil) {
+        node.layoutDelegate = self;
+      }
+      return node;
+    };
+  }
+
+  ASDataControllerCellNodeBlock block = [_asyncDataSource tableView:self nodeBlockForRowAtIndexPath:indexPath];
+  __weak __typeof__(self) weakSelf = self;
+  ASDataControllerCellNodeBlock configuredNodeBlock = ^{
```

Comment:
This method is not completely symmetrical with the Collection view version. Unless it is important they're different, make them the same, So that a future re-factor that enables codesharing does not spend time considering the significance of the differences